assets/css/home.css

```diff
@@ -81,3 +81,21 @@ body.is-os-other #quick_dl_container a.os-other {
   transition: 0.4s;
   opacity: 0.9;
 }
+
+figure {
```

Comment:
I'd prefer a separate class for this image instead of styling the whole figure.

Reply:
Note sure I know how to do that properly. Can you give an example?

Reply:
For the figure in question add a class attribute: `class="sth"` then use the sth class in css

Reply:
To have the style for the figure part of the class, you mean? Would we not do that for all figures though? I didn't want to change too much with this PR (there is also https://github.com/jamulussoftware/jamuluswebsite/issues/673 BTW which may be related)

Reply:
I mean if you think that it's for all figures put it into fw.css not home.css. 

I'd just assume that you may not want the exact same styling for each figure?
Maybe the center alignment is different? If you're sure that you want to apply this for each figure element, then go for it. But not in home.css

Reply:
I didn't intend this to be a global figure style change, but I'm also not sure why the figure isn't styled the same as the others unless I duplicate the the figure in home.css. Anyway, I'll change that so it just has a class to center the text.

Reply:
Probably this has to do with the order in which the css files get loaded. home.css is strictly only for what happens on the home page while fw.css is integrated everywhere.